include/Lv2Options.h

```diff
@@ -54,13 +56,21 @@
 class Lv2Options
 {
 public:
+	//! Return if an option is supported by LMMS
+	static bool isOptionSupported(LV2_URID key);
+	//! Mark option as supported
+	static void supportOption(LV2_URID key);
+
 	//! Initialize an option
-	void initOption(LV2_URID key,
-		uint32_t size,
-		LV2_URID type,
-		const void* value,
-		LV2_Options_Context context = LV2_OPTIONS_INSTANCE,
-		uint32_t subject = 0);
+	template<typename Opt, typename Arg>
+	void initOption(const Lv2UridCache& cache, Lv2UridCache::Id key, Arg&& value,
+			LV2_Options_Context context = LV2_OPTIONS_INSTANCE,
+			std::uint32_t subject = 0)
```

Comment:
Is it really necessary to pass the URID cache as a parameter, or could it be accessed normally as `Engine::getLv2Manager()->uridCache()` in this function?